docs/UserGuide.md

```diff
@@ -135,14 +173,15 @@ Format: `view INDEX`
 Example Command | Result
 -----------------|--------
 `list` <br> `view 2` | views the 2nd book in the list. 
+
 *This assumes you are tracking at least 2 books.*
```

Comment:
I feel like this line is a bit awkward, perhaps we could add it in the line above instead? Something like, "Look at the detailed information of your recorded book by using our View Command". Let's say you have two books in your list. 

Example command..... '

Reply:
Lets not use "Lets say"